cmd/nmi/main.go

```diff
@@ -114,7 +114,7 @@ func main() {
 	if err = metrics.RegisterAndExport(*prometheusPort); err != nil {
 		klog.Fatalf("Could not register and export metrics: %+v", err)
 	}
-	if err := s.Run(); err != nil {
-		klog.Fatalf("%s", err)
-	}
+
+	// Run nmi server
+	server.RunServer(s)
```

Comment:
>RunServer [](start = 8, length = 9)

Is there any reason we removed the error handling here?

Reply:
No, it is not removed. Instead, it is moved inside server.RunServer(s)  function.

Reply:
With the new change, it is back in this file:)